### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

**Comment by aliok** [comment]
cc @laurafitzgerald

**Review by odra — APPROVED** [review]

**Review by laurafitzgerald — APPROVED** [review]

**Comment by aliok** [comment]
@laurafitzgerald @odra I can't merge this PR (no write access). could you do that for me?

**Comment by laurafitzgerald** [comment]
@aliok done

**Pull request merged**
